### PR TITLE
Move questions between components

### PR DESCRIPTION
**Pull request by josepjaume**
#### :tophat: What? Why?
This allows moving questions between compatible components (with the same manifest name).

#### :pushpin: Related Issues
- Implements #30 

#### :clipboard: Subtasks
- [x] Move questions
- [x] Add locales
- [x] Send notifications
- [x] Log move
- [x] Add tests

### :camera: Screenshots (optional)
*None*

### CODE REVIEW

**Comment by josepjaume** [comment]
Hi @moustachu - I think  this is ready to be reviewed :). I've added specs for the command and a system spec (which the rest of the admin panel hasn't). 

The authors are notified via decidim internal notifications of the fact that a question has been moved, and an entry on the dashboard panel is created as well.

Is that ok for you? Waiting for the approval!

File: app/controllers/decidim/questions/admin/questions_moves_controller.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      class QuestionsMovesController < Admin::ApplicationController
+        def create
+          enforce_permission_to :merge, :questions
+
+          @form = form(Admin::QuestionsMoveForm).from_params(params)
+
+          Admin::MoveQuestions.call(@form, current_user) do
+            on(:ok) do |_question|
+              flash[:notice] = I18n.t("questions_moves.create.success", scope: "decidim.questions.admin")
+              redirect_to EngineRouter.admin_proxy(current_component).root_path
```
Comment (oriolgual) [inline app/controllers/decidim/questions/admin/questions_moves_controller.rb:15]: Maybe we should redirect to the target component? (since we're moving)

Comment (josepjaume) [reply app/controllers/decidim/questions/admin/questions_moves_controller.rb:15]: I thought a bit about it, but it doesn't make much sense - you might want to move resources to different places, or merge them (you're probably "gardening") and that would get you out of the flow.

**Comment by armandfardeau** [comment]
@josepjaume Hi and thanks for your contribution! I updated translations and version but the other parts seems good to me @moustachu.

**Review by moustachu — APPROVED** [review]
Thanks guys ! Great work !

**Pull request merged**